### ISSUE TITLE
Incompatible with YarnSpinner 2.x syntax

### ISSUE THREAD

**Issue by Pennycook** [kyperbelt/GDYarn#12]
YarnSpinner 2.0 made [some changes](https://github.com/YarnSpinnerTool/YarnSpinner/releases/tag/v2.0.0) to the syntax. For example, `[[Node]]` is now written `<<jump Node>>`.

GDYarn currently ignores such jumps (presumably because they look like custom commands).

**Comment by kyperbelt** [comment]
Yes, currently GDYarn is till 1.0 spec. I might do a gradual upgrade but maintain backward compatibility. I cannot give a time estimate of when this will be done. 

If you would like to make a contribution let me know. I still need to go back and add unit testing in order to ensure no functionality is lost so it might be challenging to move forward until then.
